scripts/run_all_benchmarks.sh

```diff
@@ -3,10 +3,9 @@
 export RUST_LOG=info
 THIS_DIR=$( dirname -- "$0"; )
 PROJECT=${1:-$THIS_DIR/..}
-RUNTIME=$PROJECT/target/release/frequency
+RUNTIME=$PROJECT/target/production/frequency
 BENCHMARK="$RUNTIME benchmark pallet "
```

Comment:
@wilwade  here the benchmarks are always run on production binary so this fixes the benchmarks